mods/ra/maps/allies-05a/allies05a.lua

```diff
@@ -240,6 +240,9 @@ InitTriggers = function()
 	end)
 
 	Trigger.OnInfiltrated(Prison, function()
+		if not greece.IsObjectiveCompleted(infWarfactory) then
+			Media.DisplayMessage("Good work! But next time skip the heroics!", "Battlefield Control")
+			greece.MarkCompletedObjective(infWarfactory) end
```

Comment:
Can you put the `end` on it's own line, please?

Reply:
Done.